app/assets/javascripts/admin/addon/routes/admin-route-map.js

```diff
@@ -209,6 +209,14 @@ export default function () {
       }
     );
 
+    this.route(
+      "adminFlags",
```

Comment:
I think this fine for now but pending questions around config area we may move it to /admin/config/:area route, shouldn't be too hard to recalibrate at this point :)